schema-registry/rest-service/src/test/java/com/hortonworks/registries/schemaregistry/avro/AvroSchemaRegistryClientTest.java

```diff
@@ -96,7 +96,9 @@
 
     @CustomParameterizedRunner.Parameters
     public static Iterable<SchemaRegistryTestProfileType> profiles() {
-        return Arrays.asList(SchemaRegistryTestProfileType.DEFAULT, SchemaRegistryTestProfileType.SSL);
+        return Arrays.asList(SchemaRegistryTestProfileType.DEFAULT,
+                             SchemaRegistryTestProfileType.SSL,
+                             SchemaRegistryTestProfileType.SSL_WITH_SERVER_AUTH);
```

Comment:
nit: SSL_WITH_SERVER_AUTH sounds a bit confusing to me. Do you want to change to it to ONE_WAY_SSL?

Reply:
The profile I introduced requires server authentication but client doesn't and I wanted to add another profile where client requires authentication but server doesn't. If you think its ok to only have the former we can change it to ONE_WAY_SSL, if not I will add SSL_WITH_CLIENT_AUTH profile.